### ISSUE TITLE
All variables in dotenv file are converted to string type

### ISSUE THREAD

**Issue by yugasun** [serverless/components#796]
I found `serverless cli` using [dotenv](https://github.com/motdotla/dotenv) to parse `.env` file, but `dotenv` will convert all variables to `string`, if I config `number` type in `.env` for example `PORT=3306`, then my mysql connection will throw `port unknow error`.

Ralative issue: https://github.com/motdotla/dotenv/issues/51

**Comment by medikoo** [comment]
@yugasun by design env variables can only be strings, you cannot set at CLI level a number value, it's also only strings that you can store at  `process.env` in Node.js (if you try to assign _number_, it'll be coerced to string).

Also recommended (and imo the only valid) use of `dotenv` is to at begin of program initialization transport all env vars from `.env` to `process.env`, and after that read them only from `process.env` (technically weather we're relying on `dotenv` or not, should be transparent to our app logic. It's the `process.env` that should be treated as the only source of truth)

In light of above, real fix for that issue, is to coerce `PORT` value to number before passing  it to mysql connection constructor.

**Comment by medikoo** [comment]
Yes, technically you can store stringified JSON as env var, and then resolve it when needed.

I'm not sure exactly what problem we're trying to solve (?) Where is this mysql connection setup which raises issues?

**Comment by yugasun** [comment]
@medikoo It's just an example, the key point is user want to keep the type of variables in .env config.
